msvc_build.bat

```diff
@@ -32,19 +32,22 @@ REM    "msvc_build.bat clean" remove all files in msvc.* folders and the folders
 
 REM BUILD_64 and BUILD_FOR_WINSTORE are used in CMakeLists.txt
 SET BUILD_FOR_WINSTORE=OFF
-SET BUILD_64=ON
-
-SET BUILD_FOLDER=msvc.build
-SET INSTALL_FOLDER=msvc.install
-SET ARCH=x64
-SET GENERATOR_NAME="Visual Studio 15 2017 Win64"
+SET "BUILD_FOLDER=msvc.build"
+SET "INSTALL_FOLDER=msvc.install"
```

Comment:
I don't really know much about Windows command line scripting, but what is the purpose of having quotes here? It looks like the variable will be anyway assigned correctly, and I would prefer to be very cautious with using quotes in `bat` scripting as cmd.exe seems to sometimes add those quotes to a variable value itself.

Reply:
Quoting the full command argument (not the value itself) is indeed not required if the value itself doesn't contain spaces. Once the value contains spaces you have two options:
1. enclose the value itself into quotes; but now variable substitution will also include those quotes as part of the value
2. enclose the full command argument in quotes; the variable will then still be assigned the correct value, but without the enclosing quotes.

So strictly speaking they aren't required here, but the are at L74, 80 etc where both variables are combined into the BUILD_FOLDER.

Note that Louis based his quoting on https://github.com/musescore/MuseScore/pull/4427 which was made at a point in time where the values themselves used to be quoted.
See specifically the sample output of both variants at https://github.com/musescore/MuseScore/pull/4427#issuecomment-448641620

Quoting the full set command here as well is then a more future proof and consistency measure.

Reply:
I indeed just copied #4427 and tested that it works on my system.